workbench/test/test_scenarios.py

```diff
@@ -1,26 +1,33 @@
-"""Test that all scenarios render successfully."""
+"""
+Test that all scenarios render successfully.
+"""
 
 import unittest
 
-import ddt
 import lxml.html
+import pytest
+from xblock.test.tools import assert_equals
+
 from django.core.urlresolvers import reverse
 from django.test.client import Client
-from xblock.test.tools import assert_equals
 
 from workbench import scenarios
 from workbench.runtime_util import reset_global_state
 
+pytestmark = pytest.mark.django_db  # pylint: disable=invalid-name
```

Comment:
Is this actually being used?  Same question on the other files defining it.

Reply:
Yes, from what I read it's the best practice way to get db access under pytest_django so that we're explicit about which tests hit the database. If we know of a better way of doing this I'm happy to change the pattern, though.

Reply:
Ah, I'd missed the docs on applying a mark to an entire module.  Good to know.